cmd/src/users_clean.go

```diff
@@ -0,0 +1,188 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/jedib0t/go-pretty/v6/table"
+
+	"github.com/sourcegraph/src-cli/internal/api"
+)
+
+func init() {
+	usage := `
+This command removes users from a Sourcegraph instance who have been inactive for 60 or more days. Admin accounts are omitted by default.
+	
+Examples:
+
+	$ src users clean -days 182
+	
+	$ src users clean -removeAdmin -removeNeverActive 
+`
+
+	flagSet := flag.NewFlagSet("clean", flag.ExitOnError)
+	usageFunc := func() {
+		fmt.Fprintf(flag.CommandLine.Output(), "Usage of 'src users %s':\n", flagSet.Name())
+		flagSet.PrintDefaults()
+		fmt.Println(usage)
+	}
+	var (
+		daysToDelete       = flagSet.Int("days", 60, "Days threshold on which to remove users, must be 60 days or greater and defaults to this value ")
+		removeAdmin        = flagSet.Bool("removeAdmin", false, "clean admin accounts")
+		removeNoLastActive = flagSet.Bool("removeNeverActive", false, "removes users with null lastActive value")
+		skipConfirmation   = flagSet.Bool("force", false, "skips user confirmation step allowing programmatic use")
+		apiFlags           = api.NewFlags(flagSet)
+	)
+
+	handler := func(args []string) error {
+		if err := flagSet.Parse(args); err != nil {
+			return err
+		}
+		if *daysToDelete < 60 {
+			fmt.Println("-days flag must be set to 60 or greater")
+			return nil
+		}
+
+		ctx := context.Background()
+		client := cfg.apiClient(apiFlags, flagSet.Output())
+
+		query := `
+query Users() {
+	users() {
+		nodes {
+			...UserFields
+		}
+	}
+}
+` + userFragment
+
+		// get users to delete
+		var result struct {
+			Users struct {
+				Nodes []User
+			}
+		}
+		if ok, err := client.NewRequest(query, nil).Do(ctx, &result); err != nil || !ok {
+			return err
+		}
+
+		usersToDelete := make([]UserToDelete, 0)
+		for _, user := range result.Users.Nodes {
+			daysSinceLastUse, wasLastActive, err := computeDaysSinceLastUse(user)
+			if err != nil {
+				return err
+			}
+			if !wasLastActive && !*removeNoLastActive {
+				continue
+			}
+			if !*removeAdmin && user.SiteAdmin {
+				continue
+			}
+			if daysSinceLastUse <= *daysToDelete && wasLastActive {
+				continue
+			}
+			deleteUser := UserToDelete{user, daysSinceLastUse}
+
+			usersToDelete = append(usersToDelete, deleteUser)
+		}
+
+		if *skipConfirmation {
+			for _, user := range usersToDelete {
```

Comment:
```suggestion
			for _, user := range usersToDelete {
			  if user.ID == currentUser.ID {
			  	continue
			  }
```

Reply:
@mrnugget this was bugging me so I took care of it in the last commit here: https://github.com/sourcegraph/src-cli/pull/826/commits/817d78e41d851981cabf226406b603932feac41f